q2_fondue/scraper.py

```diff
@@ -328,6 +377,9 @@ def scrape_collection(
         # get text
         try:
             str_text = zot.fulltext_item(attach_key)['content']
+            # remove character frequently placed before soft hyphen, see
```

Comment:
```suggestion
            # remove the soft hyphen, see
```

That will remove hyphens themselves, no?

Reply:
yes, that's right. thanks for the comment 👍🏼